packages/destination-actions/src/destinations/fullstory/index.ts

```diff
@@ -41,6 +41,9 @@ const destination: DestinationDefinition<Settings> = {
   },
 
   onDelete: async (request, { settings, payload }) => {
+    if (payload.userId === null || payload.userId === undefined) {
+      return
```

Comment:
Maybe a style thing, but do would it be a good idea to explicitly return `undefined`?

Reply:
@patrick-fs  I dug into `onDelete` further, and the return type is `MaybePromise<any>`. I couldn't find guidance on what to return, and the only precedent I can find is the [Friendbuy destination returning `true`](https://github.com/cowpaths/segmentio-action-destinations/blob/2a01b75cf685d5d5e4dbb85c71ba692966b1566f/packages/destination-actions/src/destinations/friendbuy/index.ts#L48) given a falsy `userId`. It's probably not worth making an HTTP request for an empty string `userId` either. How would you feel about us changing this to:

```
if (!payload.userId) {
  return true;
}
```

Reply:
Thinking a bit more on this, it feels like returning `true` could mask a configuration or usage issue. I'll connect with Josh to see if we can get guidance from Segment. Thanks for pointing this out.

Reply:
@patrick-fs We received [guidance from Segment](https://fullstory.slack.com/archives/C02F3CV9E1F/p1657033438496239?thread_ts=1657032273.228549&cid=C02F3CV9E1F) that:

- `userId` should not be falsy here since `userId` is required to kick off a deletion
- Throwing an `IntegrationError` makes sense

I've pushed a change reflecting this guidance.

Reply:
Great! Thanks for following up with them.